fga-eps-mds/2020.1-stay-safe-front-end
# Add information about features

## Issue 
Resolves #67 


## Description

Add information about existing features to increase user understanding about the app. Like a caption for Heat Map and a information button on NeighborhoodReview screen.

## Checklist:



## How to validate?

Select "Cidade" on Home screen to see the info button;
Connect NeighborhoodReview screen to another screen to see the info button;
